scripts/geopmpy/io.py

```diff
@@ -877,7 +877,10 @@ def load_reports(self, reports, dir_name, dir_cache, verbose, do_cache):
                     self._unmarked_reports_df = pandas.read_hdf(self._report_h5_name, 'unmarked_report')
                 except:
                     self._unmarked_reports_df = self._reports_df.loc[self._reports_df['region'] == 'unmarked-region']
-                self._epoch_reports_df = pandas.read_hdf(self._report_h5_name, 'epoch_report')
+                try:
+                    self._epoch_reports_df = pandas.read_hdf(self._report_h5_name, 'epoch_report')
+                except:
+                    pass
```

Comment:
Can we be more specific about what exception is being handled here, and maybe print a warning?

Reply:
Issue opened to deal with this everywhere: https://github.com/geopm/geopm/issues/3193